deriv.com.conf

```diff
@@ -25,6 +25,10 @@ server {
         return 301 https://deriv.com/contact_us/;
     }
 
+    location ~ "^/([\w]{2}|zh-tw|zh-cn)/careers" {
+        return 301 https://deriv.com/careers/;
```

Comment:
This doesn't cover for `deriv.me` right?

Reply:
It doesn't directly send to deriv.me, but if deriv.com is not available in the region automatically user will be redirected to deriv.me, no?

Reply:
I mean it won't redirect for deriv.me/pt/careers ---> deriv.me/careers

Reply:
I think it should, since this config is not specific for any domains...so it should redirect this way deriv.me/pt/careers ---> deriv.com/careers ---> deriv.me/careers. @afshinpaydar-binary please confirm ya 🙏

Reply:
Good point, let me fix that.

Reply:
Now, it will be redirect based on `$http_host`, for example:
```
deriv.me/pt/careers ---> deriv.me/careers
deriv.com/pt/careers ---> deriv.com/careers
deriv.app/pt/careers ---> deriv.app/careers
```

Reply:
Yes true, I suppose we also need the same config for those 2 redirects 🙏

Reply:
Changes applied.